src/libs/Localize/index.ts

```diff
@@ -59,12 +59,7 @@ function init() {
  * phrase and stores the translated value in the cache and returns
  * the translated value.
  */
-function getTranslatedPhrase<TKey extends TranslationPaths>(
-    language: 'en' | 'es' | 'es-ES',
-    phraseKey: TKey,
-    fallbackLanguage: 'en' | 'es' | null,
-    ...parameters: TranslationParameters<TKey>
-): string | null {
+function getTranslatedPhrase<TKey extends TranslationPaths>(language: 'en' | 'es', phraseKey: TKey, ...parameters: TranslationParameters<TKey>): string | null {
```

Comment:
From `src/types/onyx/Locale`:

```suggestion
function getTranslatedPhrase<TKey extends TranslationPaths>(language: Locale, phraseKey: TKey, ...parameters: TranslationParameters<TKey>): string | null {
```